pl/lessons/basics/pipe-operator.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: page
+title: Operator Potoku
+category: basics
+order: 6 
+lang: pl
+---
+
+Operator potoku `|>` przekazuje wynik jednego wyrażenia jako pierwszy parametr następnego wyrażenia.
+
+## Spis treści
+
+- [Wprowadzenie](#wprowadzenie)
+- [Przykłady](#przyklady)
+- [Najlepsze praktyki](#najlepsze-praktyki)
+
+## Wprowadzenie
+
+Programowanie może być chaotyczne. Tak chaotyczne, że kolejne, zagnieżdżone wywołania funkcji mogą stać się bardzo trudne do śledzenia i zrozumienia. Weźmy pod uwagę poniższy przykład, przedstawiający zagnieżdżone funkcje:
+
+```elixir
+foo(bar(baz(new_function(other_function()))))
+```
+
+Najpierw przekazujemy wartość funkcji `other_function/1` do `new_function/1`, następnie `new_function/1` do `baz/1`, z `baz/1` do `bar/1`, i ostatecznie wynik funkcji `bar/1` do `foo/1`. Elixir przyjmuje pragmatyczne podejście do tego składniowego chaosu dostarczając operator potoku. Operator ten, `|>`, *przyjmuje wynik jednego wyrażenia i przekazuje go dalej*. Przyjrzyjmy się wcześniejszemu fragmentowi kodu, przepisanemu z wykorzystaniem operatora potoku:
```

Comment:
is:
chaosu dostarczając operator
should be:
chaosu**,** dostarczając operator